output/paper/TemplateTex.log

```diff
@@ -0,0 +1,394 @@
+This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2021) (preloaded format=pdflatex 2022.2.4)  24 AUG 2022 12:25
```

Comment:
Maybe gitignore this LOG file? I think the info anyway ends up in the `sconscript.log`.

Reply:
This was already in the gitignore but had to be removed from the repo. Done in https://github.com/JMSLab/Template/pull/56/commits/1aafefda0f0c9a82a09ae0cffa06b458611081d6.